src/components/Tabs/Tabs.tsx

```diff
@@ -245,9 +247,14 @@ class TabsInner extends PureComponent<CombinedProps, State> {
   };
 
   // eslint-disable-next-line @shopify/react-no-multiple-render-methods
-  private renderTabMarkup = (tab: TabDescriptor, index: number) => {
+  private renderTabMarkup = (
+    tab: TabDescriptor,
+    index: number,
+    children: React.ReactNode,
```

Comment:
Class components will always have access to props (`this.props`) so it's not required to pass them as props 😄

Reply:
Thank you for catching this!